README.md

```diff
@@ -1,77 +1,44 @@
+## Prerequisites
 
-## build apisix RPM for CentOS 7
+- Docker
+- fpm
+- Make
```

Comment:
I think you can write an example about how to install those packages.

eg: on the Ubuntu OS.

Reply:
ping @idbeta

Reply:
I understand what you mean, I think this can be added later, let me add it to the todo list. I did not write these examples because I referred to Kong's practice, they did not write.

Reply:
ok, got it. please create an issue, we can fix this comment later